backend.py

```diff
@@ -75,7 +79,7 @@ def view(filename):
 		with open("./Notes/" + filename, "r+") as f:
 			s = f.read()
 	else:
-		s = "File does not exist"
+		s = "false"

```

Comment:
Have you considered returning False and writing something like
```
if not search_for_filename(file):
    print("Whatever string you'd like")
```

If we returned the False type then you would have full control of this string on your end.

Reply:
Frontend.py looks good to me. I like the new added fail statements like in help.